README.md

```diff
@@ -53,8 +53,8 @@ See [LICENSE][12] for further details.
 
 [1]: <https://secure.travis-ci.org/MushroomObserver/mushroom-observer.png>
 [2]: <http://travis-ci.org/MushroomObserver/mushroom-observer>
-[3]: <https://coveralls.io/repos/MushroomObserver/mushroom-observer/badge.png?branch=master>
-[4]: <https://coveralls.io/r/MushroomObserver/mushroom-observer?branch=master>
+[3]: <https://coveralls.io/repos/MushroomObserver/mushroom-observer/badge.png?branch=main>
+[4]: <https://coveralls.io/r/MushroomObserver/mushroom-observer?branch=main>
 [5]: <https://codeclimate.com/github/MushroomObserver/mushroom-observer.png>
 [6]: <https://codeclimate.com/github/MushroomObserver/mushroom-observer>
-[12]: <https://github.com/MushroomObserver/mushroom-observer/blob/master/LICENSE>
+[12]: <https://github.com/MushroomObserver/mushroom-observer/blob/main/LICENSE>
```

Comment:
Would relative links be better?  See https://github.blog/2013-01-31-relative-links-in-markup-files/
```suggestion
[12]: <LICENSE>
```